docs/resources/database.md

```diff
@@ -28,6 +28,7 @@ resource "materialize_database" "example_database" {
 ### Optional
 
 - `comment` (String) **Public Preview** Comment on an object in the database.
+- `no_public_schema` (Boolean) When set to true, the default 'public' schema will be dropped from the database.
```

Comment:
I'd flip this around and make it `public_schema` or `create_public_schema` — in general I think it's much easier to reason about things when all boolean parameters are positive rather than negative.